src/style.css

```
@@ -1,3 +1,93 @@
+:root {
+  --active-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  width: 100%;
+  background: #fff;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__list {
+  align-items: center;
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.logo__image {
+  display: flex;
+  width: 40px;
+  height: 40px;
+  image-rendering: pixelated;
+}
+
+.nav__link {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+  position: relative;
+}
+
+.nav__link.is-active {
+  color: var(--active-color);
+  position: relative;
+}
+
+.nav__link:hover {
+  color: var(--active-color);
+}
+
+.nav__link::after {
+  content: '';
+  transition: all 0.3s;
+  display: none;
+}
+
+.nav__link.is-active::after {
```

```suggestion
.is-active::after {
```
Same - Redundant combined selector